internal/featuresupport/features_olm_operators.go

```diff
@@ -591,3 +591,37 @@ func (f *OpenShiftAIFeature) getFeatureActiveLevel(cluster *common.Cluster, _ *m
 	}
 	return activeLevelNotActive
 }
+
+// AuthorinoFeature describes the support for the Authorino operator.
+type AuthorinoFeature struct{}
+
+func (f *AuthorinoFeature) New() SupportLevelFeature {
+	return &AuthorinoFeature{}
+}
+
+func (f *AuthorinoFeature) getId() models.FeatureSupportLevelID {
+	return models.FeatureSupportLevelIDAUTHORINO
+}
+
+func (f *AuthorinoFeature) GetName() string {
+	return "Authorino"
+}
```

Comment:
the operator name is different from the feature name? 
```go
func (o *operator) GetFullName() string {
	return "OpenShift AI"
}
```

Reply:
That is a copy & paste error. Both should be equal.

Reply:
Done.